pycbc/workflow/jobsetup.py

```diff
@@ -778,6 +779,128 @@ def zero_pad_data_extend(self, job_data_seg, curr_seg):
+    pycbc_multi_inspiral executable.
+    """
+    current_retention_level = Executable.MERGED_TRIGGERS
+    #file_input_options = ['--gating-file']
+    def __init__(self, cp, name, universe=None, ifo=None, injection_file=None,
+                 gate_files=None, out_dir=None, tags=None):
+        if tags is None:
+            tags = []
+        super(PyCBCMultiInspiralExecutable, self).__init__(cp, name, universe,
+                ifo, out_dir=out_dir, tags=tags)
+        self.injection_file = injection_file
+        self.data_seg = segments.segment(int(cp.get('workflow', 'start-time')),
+                                         int(cp.get('workflow', 'end-time')))
+        self.num_threads = 1
+
+    def create_node(self, data_seg, valid_seg, parent=None, inj_file=None,
+                    dfParents=None, bankVetoBank=None, ipn_file=None,
+                    slide=None, tags=None):
+        if tags is None:
+            tags = []
+        node = Node(self)
+
+        if not dfParents:
+            raise ValueError("%s must be supplied with frame or cache files"
+                              % self.name)
+
+        # If doing single IFO search, make sure slides are disabled
+        if len(self.ifo_list) < 2 and \
+                (node.get_opt('--do-short-slides') is not None or \
+                 node.get_opt('--short-slide-offset') is not None):
+            raise ValueError("Cannot run with time slides in a single IFO "
+                             "configuration! Please edit your configuration "
+                             "file accordingly.")
+
+        # Set instuments
+        node.add_opt("--instruments", " ".join(self.ifo_list))
+
+        pad_data = self.get_opt('pad-data')
+        if pad_data is None:
+            raise ValueError("The option pad-data is a required option of "
+                             "%s. Please check the ini file." % self.name)
+
+        # Feed in bank_veto_bank.xml
+        if self.cp.has_option('inspiral', 'do-bank-veto'):
+            if not bankVetoBank:
+                raise ValueError("%s must be given a bank veto file if the "
+                                 "argument 'do-bank-veto' is given"
+                                 % self.name)
+            node.add_input_opt('--bank-veto-templates', bankVetoBank)
+
+        # Set time options
+        node.add_opt('--gps-start-time', data_seg[0] + int(pad_data))
+        node.add_opt('--gps-end-time', data_seg[1] - int(pad_data))
+        node.add_opt('--trig-start-time', valid_seg[0])
+        node.add_opt('--trig-end-time', valid_seg[1])
+
+        node.add_profile('condor', 'request_cpus', self.num_threads)
+
+        # Set the input and output files
+        node.new_output_file_opt(data_seg, '.hdf', '--output',
+                                 tags=tags, store_file=self.retain_files)
+        node.add_input_opt('--bank-file', parent, )
+
+        # TODO: isn't there a cleaner way of doing this?
```

Comment:
Shouldn't you be using `--frame-files` like `PyCBCInspiral` ... That might need a big "FIXME" in flashing lights, as this would prevent the code running in places like OSG.

Reply:
I have no idea with this. Maybe @pannarale  can help with this?

Reply:
@pannarale Is there some sort of to-do list for GRB development? Fixing this may not belong in this pull request, but this shouldn't get lost.